src/seedu/addressbook/data/person/PostalCode.java

```diff
@@ -0,0 +1,29 @@
+package seedu.addressbook.data.person;
+
+public class PostalCode{
+	
+	private String postalCodeNum;
+	
+	public PostalCode(String postalCodeNum) {
+		this.postalCodeNum = postalCodeNum;
+	}
+	
+	public String getpostalCodeNum() {
+		return postalCodeNum;
+	}
```

Comment:
Please fix the indentation and change the tab to spaces in Eclipse, see [here](http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs)